content/Mobile_Dev/WebAuthn/WebAuthn_Primer.adoc

```diff
@@ -0,0 +1,54 @@
+* Guide developers on how to deploy an example WebAuthn application.
+
+=== Where can I learn more about WebAuthn?
+Below are a variety of resources for you to continue to learn more about WebAuthn 
+
+* link:/WebAuthn/WebAuthn_Developer_Guide/[Yubico WebAuthn Developer Guide] - Overview of a variety of topics to help you become familiar with WebAuthn concepts
+* link:https://www.edx.org/course/introduction-to-web-authentication[Introduction to WebAuthn] - Course developed by W3C and Yubico meant to introduce developers to the WebAuthn API and concepts
+* link:https://www.w3.org/TR/webauthn-2/[WebAuthn specification] - W3C specification for the WebAuthn protocol.
+* link:https://github.com/YubicoLabs/WebAuthnKit[WebAuthn Starter Kit] - Open source project for you to create your own test environment
+
+=== Important concepts to consider
+The following subsections will outline considerations to take during the implementation of your application. While the concepts below aren’t inherently specific to mobile applications with WebAuthn, it’s worth reinforcing them to help you frame your implementation and UX strategy.
+
+==== Platform vs Cross Platform authenticators
+The WebAuthn standard supports two types of authenticators:
+
+Platform authenticators, also known as internal authenticators, such as a fingerprint scanner built into a smartphone. Platform authenticators are limited to authenticating a user via a specific device (in the case of Windows Hello, the laptop running it). To support platform authenticators, a device must include both:
+
+* A Trusted Platform Module (TPM) security chip to handle public and private keys. Most recent business-grade laptops, desktops, and smartphones have TPMs.
+* A camera or biometric reader that supports the types of input required; for example, to scan a fingerprint, a fingerprint reader is required. Note that a biometric sensor is not required for a platform authenticator. The point is that the authenticator must support the type of input required, which could just as well be PIN for user verification, or indeed, in the case where there is no user verification at all, the TPM would be used as a U2F style second factor authenticator.
+
+Roaming authenticators, also known as cross-platform or external authenticators, such as a hardware security key. These authenticators can be used with a laptop or mobile device, etc., and are thus cross-platform. This characteristic enables their use to establish a secure source for verifying the user’s identity (the "root of trust") and for delegating trust to specific devices in the user’s control. The user can therefore authorize other devices as needed ("bootstrap" them). Should one of those other devices be lost or become corrupted, the roaming authenticator is used to authorize a new device. For these reasons, it is a roaming authenticator such as a security key that should be the primary authenticator.
```

Comment:
Consider bolding `Roaming authenticators`

Reply:
Added in newest commit